apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -378,7 +378,9 @@ type HttpManagement struct {
 	// When enabled, HTTP will be used instead of JMX for management connectivity between Cassandra
 	// and Reaper. In future, this will be true by default
 	// +kubebuilder:default=false
-	Enabled *bool `json:"enabled"`
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
[thought] I would be careful changing that, since the comment says it might default to `true` in the future.

The problem with a raw boolean is that in some case it becomes impossible to differentiate between "unset" and "explicitly set to false".